backend/db/migrate/20240229135718_migrate_study_learning_paths.rb

```diff
@@ -1,43 +1,123 @@
 class MigrateStudyLearningPaths < ActiveRecord::Migration[6.1]
```

Comment:
All data here per [this doc](https://docs.google.com/document/d/1nzGtI-DA2mDzNRuns2ZlSocvC7uH21qMmn5FT6CX5Os/edit)